Umee-Bots/uToken-Exchange-Rate-Drop/src/agent.spec.ts

```diff
@@ -0,0 +1,319 @@
+import { HandleBlock, Initialize } from "forta-agent";
+import { provideHandleBlock, provideInitialize } from "./agent";
+import { createAddress, TestBlockEvent } from "forta-agent-tools/lib/tests.utils";
+import { when, resetAllWhenMocks } from "jest-when";
+import { createFinding, calculateSeverity, AgentConfig } from "./utils";
+import BigNumber from "bignumber.js";
+
+const mockFetcher = {
+  getUnderlyingAssetAddress: jest.fn(),
+  getPrice: jest.fn(),
+  getReserveNormalizedIncome: jest.fn(),
+};
+
+const testUnderlyingAssetAddresses = [createAddress("0xua1"), createAddress("0xua2"), createAddress("0xua3")];
```

Comment:
```suggestion
const testUnderlyingAssetAddresses = [createAddress("0x0a1"), createAddress("0x0a2"), createAddress("0x0a3")];
```
Minor, but better to keep addresses as hex strings.